C-Plus-Plus/other/count_of_set_bit_using_lookup_tech.cpp

```diff
@@ -0,0 +1,47 @@
+/*A CPP implementation of count of set bits in an integer using a LookUp 
+table technique.
+
+Piyush Kumar
+
+ */
+
+#include <bits/stdc++.h>
+#include <iostream>
+#include <algorithm>
+#include <vector>
```

Comment:
No need to add these header files, as you have already included the  **<bits/stdc++.h>**.